test/fixtures/test-runner/plan/nested-subtests.mjs

```diff
@@ -0,0 +1,14 @@
+import test from 'node:test';
+
+test('deeply nested tests', async (t) => {
+  t.plan(1);
+
+  await t.test('level 1', async (t) => {
```

Comment:
```suggestion
  t.test('level 1', async (t) => {
```

Reply:
see https://github.com/nodejs/node/pull/56664

Reply:
It might be worth keeping the `await`s so that the commit can be backported.